model/godleyTableWindow.cc

```diff
@@ -162,7 +162,6 @@ namespace minsky
     if (!godleyIcon) return;
     CairoSave cs(cairo);
     cairo_scale(cairo,zoomFactor,zoomFactor);
-    ZoomablePango::zoomFactor=zoomFactor;
```

Comment:
I do think this solves the problem. ZoomablePango was a solution to a design problem in Pango that has now been corrected. But a ZoomablePango with zoomfactor=1 is just a regular Pango, so we should retire the whole class, to keep the code tidy.